docs/dev/new_check_howto.md

```diff
@@ -274,6 +274,72 @@ The check is almost done. Let's add the final touches.
 
 ## Final touches
 
+### Enhance the configuration file
+
+Add a set of special words to the check `conf.yaml.example` configuration file in form of YAML comment strings. 
```

Comment:
Instead of `commands`, we should call then `tags` maybe even `descriptive tags` or `documentation tags` to be more explicit. `tags` is the term used in the [javadoc spec](http://www.oracle.com/technetwork/articles/java/index-137868.html) so it will be easier for people to understand it.

Reply:
👎 to use `tags`, it has a very specific meaning in our vocabulary. `Commands` is used in other documentation tools like [Doxygen](https://www.stack.nl/~dimitri/doxygen/manual/commands.html)